src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Windows.cs

```diff
@@ -802,7 +802,7 @@ public static SocketError Select(IList checkRead, IList checkWrite, IList checkE
             return (SocketError)socketCount;
         }
 
-        public static SocketError Shutdown(SafeCloseSocket handle, SocketShutdown how)
+        public static SocketError Shutdown(SafeCloseSocket handle, bool isConnected, SocketShutdown how)
         {
             SocketError err = Interop.Winsock.shutdown(handle, (int)how);
```

Comment:
This should make build do a warning since isConnected isn't used. Please add a comment to clarify why it isn't used for Windows.